hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/RestEntity.java

```diff
@@ -71,6 +72,9 @@ public RestEntity() {
         CanonicalPath path = CanonicalPath.fromPartiallyUntypedString(getPath(uriInfo, "/treeHash".length()),
                 getTenantPath(), AbstractElement.class);
 
+        if (path.getSegment().getElementType() == SegmentType.e) {
```

Comment:
Another non-synced type is the tenant.

A generic test for this, would I guess look like:

```java
Syncable.class.isAssignableFrom(Inventory.types().byPath(path).getElementType())
```